### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by dave-mills**
Merging all changes into Master.

This is a pretty messy Pull Request, as it brings in everything done since December. The goal is to have a stable base from which to run the pilot of the system. (a proper 'beta' version).

### CODE REVIEW

File: Mysql/insert_test_forms.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- Dumping data for table ukraine.forms: ~1 rows (approximately)
+/*!40000 ALTER TABLE `forms` DISABLE KEYS */;
+INSERT INTO `forms` (`id`, `title`, `xform_id`) VALUES
+    ('agRjX9kiP7TSU45B8aLsUN', 'Iodine Survey - Building Listing', 'ukraine_iodine_building_listing');
```
Comment (luciafalcinelli) [inline Mysql/insert_test_forms.sql]: In my kobotoolbox the id for this form is different, probably I have an old version. In any case are you sure is correct?

File: Mysql/select_items.sql [hunk 2]
```diff
@@ -0,0 +1,139 @@
+#salt_sampled Level
+#total number of salt samples collected 
+
+SELECT
```
Comment (luciafalcinelli) [inline Mysql/select_items.sql]: I used this file to create the summaries view, so we don't need it anymore

File: Mysql/views/district_summary.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
+ 
+
+SELECT 
+       `clusters`.`smd_id` AS `district_id`,
+       clusters.region_id as region_id,
```
Comment (luciafalcinelli) [inline Mysql/views/district_summary.sql]: Should be `clusters`.`region_id` as `region_id`, the quotation marks are missing

File: Mysql/insert_test_forms.sql (hunk 1 above)
Comment (dave-mills) [reply Mysql/insert_test_forms.sql]: This is the id for the one attached to the ukraine_iodine account in kobo humanitarian. It'll be different if you've uploaded a version into your account. 

But I should update this script with the main survey form and pilot test forms soon...

File: Mysql/views/district_summary.sql (hunk 3 above)
Comment (dave-mills) [reply Mysql/views/district_summary.sql]: Ha, nicely caught. Not a syntax error (` quotes are only essential when using strings with dashes or ones involving sql keywords), but I should be consistent here... updating...

File: Mysql/select_items.sql (hunk 2 above)
Comment (dave-mills) [reply Mysql/select_items.sql]: 👍 deleting

**Pull request merged**